data/kahet/beyond patir assets.txt

```diff
@@ -0,0 +1,2460 @@
+# Copyright (c) 2024 by Tommaso Becca
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+phrase "magic asteroid"
+	word
+		"???"
+
+fleet "Magic Asteroid"
+	government "Magic Asteroid"
+	names "magic asteroid"
+	cargo 1
+	personality
+		plunders harvests surveillance mute
+	variant 11
+		"Asteroid Large 1"
+	variant 11
+		"Asteroid Large 2"
+	variant 11
+		"Asteroid Large 3"
+	variant 11
+		"Asteroid Large 4"
+	variant 11
+		"Asteroid Large 5"
+	variant 12
+		"Asteroid Medium"
+	variant 13
+		"Asteroid Young"
+	variant 13
+		"Asteroid Young 2"
+	variant 13
+		"Asteroid Young 3"
+	variant 13
+		"Asteroid Young 4"
+
+ship "Asteroid Large 1"
+	"display name" "???"
+	"noun" "asteroid"
+	sprite "ship/asteroid adult"
+	attributes
+		category "Transport"
+		"uncapturable" 1
+		"inscrutable" 1
+		"cost" 1350000000
+		"hull" 136561
+		"required crew" 1
+		"bunks" 1
+		"mass" 1174
+		"drag" 70.4
+		"heat dissipation" 10
+		"fuel capacity" 1000
+		"energy capacity" 1
+		"fuel generation" 0.3
+		"cargo space" 40
+		"outfit space" 390
+		"weapon capacity" 150
+		"engine capacity" 95
+		"hull repair rate" 47
+		"cargo scan power" 1
+		"cargo scan efficiency" 0.001
+		"thrust" 340.2
+		"flare sprite" "effect/asteroid adult flare"
+			"frame rate" 1
+		"turn" 5931.4
+		"asteroid mount jd" 1
+		"asteroid mount am" 1
+		"asteroid mount" 1
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Asteroid Anti-Missile"
+		"Asteroid Weapon Large" 2
+		"Asteroid Jump Drive"
+		
+	engine 0 0
+		zoom 1
+		angle 0
+		over
+	gun 0 0 "Asteroid Weapon Large"
+	gun 0 0 "Asteroid Weapon Large"
+	turret 0 0 "Asteroid Anti-Missile"
+	explode "tiny explosion" 10
+	explode "small explosion" 25
+	explode "medium explosion" 25
+	explode "large explosion" 10
+	"final explode" "final explosion medium"
+
+ship "Asteroid Large 1" "Asteroid Large 2"
```

Comment:
What's the reason for making every ship a variant of the first one?

Reply:
The original idea was to have them all have the same name ("???") but since display name exists it's now unnecessary. I will change this with the rest in a few days